src/main/java/org/motechproject/newebodac/mapper/RoleMapper.java

```diff
@@ -0,0 +1,34 @@
+package org.motechproject.newebodac.mapper;
+
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.Mappings;
+import org.mapstruct.ReportingPolicy;
+import org.mapstruct.factory.Mappers;
+import org.motechproject.newebodac.domain.security.UserRole;
+import org.motechproject.newebodac.dto.RoleDto;
+
+@Mapper(uses = { UuidMapper.class },
+    unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface RoleMapper extends EntityMapper<RoleDto, UserRole> {
+
+  RoleMapper INSTANCE = Mappers.getMapper(RoleMapper.class);
+
+  RoleDto toDto(UserRole role);
```

Comment:
if you're not changing any mappers, you don't need to implement this (there is default implementation in EntityMapper), so remove it